Modelica/Blocks/Types.mo

```diff
@@ -107,11 +107,11 @@ package Types
       input String tableName "Table name";
       input String fileName "File name";
       input Real table[:, :];
-      input SI.Time startTime;
+      input Real startTimeScaled(unit = "1");
       input Integer columns[:];
       input Modelica.Blocks.Types.Smoothness smoothness;
       input Modelica.Blocks.Types.Extrapolation extrapolation;
-      input SI.Time shiftTime=0.0;
+      input Real shiftTimeScaled(unit = "1")=0.0;
```

Comment:
```suggestion
      input Real shiftTimeScaled=0.0;
```
And here as well.

Reply:
Please motivate why to do it in a way that wouldn't let someone detect the mistake to believe that the table works with true (unscaled) time.

Reply:
But that is not a mistake - the table object works with what-ever you send in as time - as far as I recall the external object wasn't rewritten to handle scaled time. ( https://github.com/modelica/ModelicaStandardLibrary/commit/70f01e3f87338629f5cffc7e739785d8226ae178 - the C-code wasn't changed, and later correction was also outside the external object)

Note that this is only relevant if people directly use the external object. If they use the block (as normal people) then unit-errors will be detected at the block boundary.

Reply:
Good, then we have a motivation.  Let's just leave this open a little longer to give more people time to react before the suggested change is applied.

Reply:
Enough time has passed, and @beutlich didn't seem to have any objections on this point.  The unit has now been removed.  Resolving.